src/components/CommunityEditionHead.tsx

```diff
@@ -0,0 +1,74 @@
+import { Locale } from 'i18n.config';
+import Head from 'next/head';
+import { ReactNode } from 'react';
+
```

Comment:
missing React in the import, this will produce this error 
![image](https://github.com/latina-in-tech/latina-in-tech.github.io/assets/12852000/18838d03-cd49-423f-828c-d0e5867dcaf0)




```suggestion
import React from 'react';
```